apis/src/pages/tournament.rs

```diff
@@ -362,3 +473,83 @@ pub fn Tournament() -> impl IntoView {
         </div>
     }
 }
+
+#[component]
+pub fn MySchedules(
+    games_hashmap: Memo<HashMap<GameId, GameResponse>>,
+    user_id: Option<Uuid>,
+) -> impl IntoView {
+    logging::log!("{:?}", user_id);
```

Comment:
As soon as we know it works correctly I would get rid of these logs/ maybe just comment them out